web/pgadmin/static/js/helpers/Notifier.jsx

```diff
@@ -182,20 +198,16 @@ class Notifier {
   }
 
   confirmDelete(title, text, onOkClick, onCancelClick,okLabel = gettext('Yes'), cancelLabel = gettext('No')){
-
-    const  extraButtons = (closeModal) => {
-      return [
-        {
-          type: 'default',
-          icon: <CheckRoundedIcon />,
-          label: okLabel,
-          onClick: ()=>{
-            onOkClick();
-            closeModal();
-          },
-          color: 'error',
-        },
-      ];
+    const extraButtons = (closeModal) => {
+      // Button configurations for specific actions like "Delete" and "Disconnect".
+      const buttonConfigs = {
+        [gettext('Delete')]: () =>
```

Comment:
the buttons config was an array, why are you creating a dictionary? You're making simple code complex.